src/helpers/logger.ts

```diff
@@ -0,0 +1,5 @@
+import { Logs } from "@ubiquity-dao/ubiquibot-logger";
```

Comment:
It does not anymore afaik. Its only purpose is to create a `diff` format which is useful for Github comments.
It would be nice to be able to directly plug Github or Supabase to log automatically.